arraycontext/context.py

```diff
@@ -504,7 +504,8 @@ def clone(self: SelfType) -> SelfType:
             "setup-only" array context "leaks" into the application.
         """
 
-    def compile(self, f: Callable[..., Any]) -> Callable[..., Any]:
+    def compile(self, f: Callable[..., Any],
+                single_version_only: bool = False) -> Callable[..., Any]:
```

Comment:
docs for single_version_only?

Reply:
Added in 5ed25a59372c098976e55adb88ae302b3d75f82b